impacket/examples/ntlmrelayx/servers/mssqlrelayserver.py

```diff
@@ -0,0 +1,294 @@
+# Impacket - Collection of Python classes for working with network protocols.
+#
+# Copyright Fortra, LLC and its affiliated companies 
+#
+# All rights reserved.
+#
+# This software is provided under a slightly modified version
+# of the Apache Software License. See the accompanying LICENSE file
+# for more information.
+#
+# Description:
+#   MS SQL Relay Server 
+#
+#   This is the MSSQL server that relays the connections
+#   to other protocols. 
+#
+#   Based on RAW Relay Server.
+#
+# Authors:
+#   Eugenie Potseluevskaya
+#
+
+import socketserver
+import random
+import string
+from threading import Thread
+
+from impacket import ntlm, tds, LOG
+from impacket.smbserver import outputToJohnFormat, writeJohnOutputToFile
+from impacket.examples.utils import parse_target
+from impacket.nt_errors import STATUS_SUCCESS
+from impacket.ntlm import NTLMAuthChallenge
+from impacket.examples.ntlmrelayx.utils.targetsutils import TargetsProcessor
+from impacket.examples.ntlmrelayx.servers.socksserver import activeConnections
+
```

Comment:
Needed to listen in IPv6 if requested so

```suggestion
from impacket.examples.utils import get_address

```